src/ingest_validation_tools/table-schemas/level-2/lightsheet.yaml

```diff
@@ -0,0 +1,24 @@
+doc_url: https://portal.hubmapconsortium.org/docs/assays/lightsheet # TODO
+fields:
+  # TODO: Overrides of Level 1 fields
+
+  # Level 2 fields:
+  # include: ../includes/acquisition_instrument.yaml
+  # include: ../includes/x_y_z.yaml
+
+  # TODO: Confirm descriptions / confirm that switching order for consistency is ok.
+  - name: number_of_antibodies
+    description: Number of antibodies
+  - name: number_of_channels
+    description: Number of fluorescent channels imaged during each cycle.
+
+  # TODO: I'm not sure number_of_z_stacks belongs:
+  #   - Do you actually mean the number of layers in the stack, or are their multiple stacks?
+  #   - Why is the z dimension different from the x or y? We don't want to duplicate what can be inferred from the data.
+
+  # TODO: For consistency, can it just be "protocols_io_doi", without "acquisition_"?
```

Comment:
I'm waiting for a response to your question from Marda.

Reply:
For changes to the text, please use the github machinery to suggest edits on the yaml file.

Reply:
I'll work on this today.